src/Field/WfsSearchInput/WfsSearchInput.jsx

```diff
@@ -0,0 +1,442 @@
+     * @type {String}
+     */
+    featureNS: PropTypes.string,
+    /**
+     * The prefix for the feature namespace.
+     * @type {String}
+     */
+    featurePrefix: PropTypes.string,
+    /**
+     * The feature type names. Required.
+     * @type {String[]}
+     */
+    featureTypes: PropTypes.arrayOf(PropTypes.string).isRequired,
+    /**
+     * SRS name. No srsName attribute will be set on geometries when this is not
+     * provided.
+     * @type {string}
+     */
+    srsName: PropTypes.string,
+    /**
+     * The output format of the response.
+     * @type {string}
+     */
+    outputFormat: PropTypes.string,
+    /**
+     * Maximum number of features to fetch.
+     * @type {number}
+     */
+    maxFeatures: PropTypes.number,
+    /**
+     * Geometry name to use in a BBOX filter.
+     * @type {string}
+     */
+    geometryName: PropTypes.string,
+    /**
+     * Optional list of property names to serialize.
+     * @type {String[]}
+     */
+    propertyNames: PropTypes.arrayOf(PropTypes.string),
+    /**
+     * Filter condition. See https://openlayers.org/en/latest/apidoc/ol.format.filter.html
+     * for more information.
+     * @type {object}
+     */
+    filter: PropTypes.object,
+    /**
+     * The ol.map to interact with on selection.
+     *
+     * @type {Object}
+     */
+    map: PropTypes.instanceOf(OlMap),
+    /**
+     * The minimal amount of characters entered in the input to start a search.
+     * @type {Number}
+     */
+    minChars: PropTypes.number,
+    /**
+     * An onFetchSuccess callback function which gets called with the
+     * successfully fetched data.
+     * Please note: if omitted only data fetch will be performed and no data
+     * will be shown afterwards!
+     * @type {function}
+     */
+    onFetchSuccess: PropTypes.func,
+    /**
+     * An onFetchError callback function which gets called if data fetch is
+     * failed.
+     * @type {function}
+     */
+    onFetchError: PropTypes.func,
+    /**
+     * Optional callback function, that will be called if 'clear' button of
+     * input field was clicked.
+     * @type {function}
+     */
+    onClearClick: PropTypes.func,
+    /**
+      * Optional callback function, that will be called before WFS search starts.
+      * Can be useful if input value manipulation is needed before (e.g. umlaut
+      * replacement `ä => ae` etc.)
+      * @type {function}
+      */
+    onBeforeSearch: PropTypes.func,
+    /**
+     * Options which are added to the fetch-POST-request. credentials is set to
+     * 'same-origin' as default but can be overwritten. See also
+     * https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch
+     * @type {object}
+     */
+    additionalFetchOptions: PropTypes.object,
+    /**
+     * Options which are passed to the constructor of the ol.format.WFS.
+     * compare: https://openlayers.org/en/latest/apidoc/ol.format.WFS.html
+     * @type {object}
```

Comment:
`object` -> `Object`.

Reply:
Link is outdated, please make use of [https://openlayers.org/en/latest/apidoc/module-ol_format_WFS.html](https://openlayers.org/en/latest/apidoc/module-ol_format_WFS.html).